docker-entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+set -o errexit
+
+# Execute whatever command was given
+exec "$@"
```

Comment:
What's the purpose of this entrypoint? It seems like setting `errexit` would only matter if we were doing other things in the script?